Formula/simple-tiles.rb

```diff
@@ -1,10 +1,9 @@
 class SimpleTiles < Formula
   desc "Image generation library for spatial data"
   homepage "https://github.com/propublica/simple-tiles"
```

Comment:
this url should match with the artifact.

Reply:
OK, fixed